elements/lisk-p2p/test/integration/penalty_malformed_peerinfo.ts

```diff
@@ -12,50 +12,51 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { platform } from 'os';
-import { P2P, events } from '../../src/index';
-import { wait } from '../utils/helpers';
-import { createNetwork, destroyNetwork } from '../utils/network_setup';
+// import { platform } from 'os';
+// import { P2P, events } from '../../src/index';
+// import { wait } from '../utils/helpers';
+// import { createNetwork, destroyNetwork } from '../utils/network_setup';
 
-const { EVENT_BAN_PEER } = events;
+// const { EVENT_BAN_PEER } = events;
 
+// Skipping as schema validation doesn't allow custom fields and supported properties are validated before applying nodeInfo.
```

Comment:
Instead of commenting out, you can simply .skip the test

Reply:
same for this